spec/controllers/users_controller_spec.rb

```diff
@@ -32,75 +33,117 @@
   end
 
   describe 'POST create' do
-    context "user input clears validations" do
+    context "when user input clears validations" do
       after { ActionMailer::Base.deliveries.clear }
 
-      it "creates a new user object" do
-        post :create, user: Fabricate.attributes_for(:user, full_name: "Pete")
-        expect(User.count).to eq(1)
-      end
+      context "when card info is valid" do
+        before do
+          charge = double('charge', successful?: true)
+          allow(StripeWrapper::Charge).to receive(:create) { charge }
+        end
+
+        it "creates a new user object" do
+          post :create, user: Fabricate.attributes_for(:user, full_name: "Pete"), stripeToken: '111'
+          expect(User.count).to eq(1)
+        end
 
-      context "email sending" do
-        let(:subject) { ActionMailer::Base.deliveries.last }
-        before { post :create, user: Fabricate.attributes_for(:user, full_name: "Pete") }
+        context "email sending" do
+          let(:subject) { ActionMailer::Base.deliveries.last }
```

Comment:
Instead of this `let(:subject) { ActionMailer::Base.deliveries.last }`

you should be able use `subject { ActionMailer::Base.deliveries.last }` or 

`subject(:mail) { ActionMailer::Base.deliveries.last }` and then in examples refer `mail` as subject.